src/components/sidebar/mine-filter-tab.tsx

```diff
@@ -61,10 +61,16 @@ function MineFilterTab({selectedTabData, oversiktType, enhettiltak}: SidevelgerP
             meta={
                 <>
                     <HelpText placement="right">
-                        {oversiktType === OversiktType.minOversikt &&
-                            'Filter som inneholder Veiledergrupper og Ufordelte brukere er ikke tilgjengelig i Min oversikt.'}
-                        {oversiktType === OversiktType.enhetensOversikt &&
-                            'Filter som inneholder Arbeidslisten og Nye brukere er ikke tilgjengelig i Enhetens oversikt.'}
+                        {(() => {
+                            switch (oversiktType) {
```

Comment:
Kunne kanskje vert trukken ut som en separat funksjon, for å øke lesbarheiten litt. Er det dessuten en spesiell grunn til at vi bruker IIFE (Immediately Invoked Function Expression) her?

Reply:
Endret, og ny versjon ligger nå i dev.

Reply:
👍